### PR TITLE
Broken menu

### PR DESCRIPTION
**Pull request by hugomg**
Fix bug #177 and some minor improvements.

### CODE REVIEW

File: chat_filter.user.js [hunk 1]
```diff
@@ -814,32 +814,17 @@ add_initializer(function(){
 
     add_custom_css([
         ".chat-room { z-index: inherit !important; }",
+
         ".chat-settings { z-index: 100 !important; }",
+        ".chat-settings { max-height: 500px; }",
```
Comment (mkody) [inline chat_filter.user.js]: I guess you can move the `max-height: 500px;` here to the brackets above.

**Comment by mkody** [comment]
It's been a while since I've done something here.
LGTM to fix the issue.
Looks like it stopped filtering too (at least for emulator and bets commands), can you confirm @hugomg ?

File: chat_filter.user.js (hunk 1 above)
Comment (hugomg) [inline chat_filter.user.js]: I thought it was more readable this way. I don't like putting more than one CSS rule per line.

**Comment by hugomg** [comment]
After a quick check, filtering the chat commands and stadium bets seems to be working. But maybe I'm missing something since I only did a quick glance. Can give some examples of messages that are not being filtered?

**Comment by mkody** [comment]
Emulator commands ([left, right, up, down, start, select, a, b, l, r](https://cloud.githubusercontent.com/assets/843787/18445132/64e9fe44-791c-11e6-9862-a159828049c6.png)) and the betting commands ([!bet before matches](https://cloud.githubusercontent.com/assets/843787/18445168/8a28daa4-791c-11e6-958e-0713e2826a05.png) and [!a, !b, !c, !d, !1, !2, !3, !-](https://cloud.githubusercontent.com/assets/843787/18445179/9b2b5228-791c-11e6-9728-2fe7a82f433a.png)).

I'm using Violentmonkey on Vivaldi, so the problem is probably there. I'll try with more browsers and extensions to see.

**Comment by mkody** [comment]
Ugh nevermind. Disabled BetterTTV and it works.
We might implement a message or something when this happen.

File: chat_filter.user.js (hunk 1 above)
Comment (mkody) [inline chat_filter.user.js]: Yup. Just saying that since I saw that `.custom_list_menu li` did have everything in one line

**Pull request merged**